### PR TITLE
[wix-storybook-utils] prepareStory - fix story structure to not include module.exports

### PR DESCRIPTION
**Pull request by mykas**
When moving to storybook 6 - whenever story file includes export default or module.exports it will expect an object with `title` defined, but we are actually using soriesOf and will use until we can migrate to new csf structure.

### CODE REVIEW

File: packages/wix-storybook-utils/src/loader/prepareStory/prepareStory.js [hunk 1]
```diff
@@ -95,7 +123,10 @@ const prepareStory = (storyConfig, sourcePath) => source =>
 
       visit(ast)({
         ExportDefaultDeclaration(path) {
-          path.node.declaration = handleExportObject(path, path.node.declaration);
+          ast.program.body.push(
```
Comment (mykas) [inline packages/wix-storybook-utils/src/loader/prepareStory/prepareStory.js:126]: look here

File: packages/wix-storybook-utils/src/loader/prepareStory/prepareStory.js [hunk 2]
```diff
@@ -107,7 +138,10 @@ const prepareStory = (storyConfig, sourcePath) => source =>
           ].every(Boolean);
 
           if (isModuleExports) {
-            path.node.expression.right = handleExportObject(path, path.node.expression.right);
+            ast.program.body.push(
```
[the remark this answers is not in the document]
Comment (mykas) [reply packages/wix-storybook-utils/src/loader/prepareStory/prepareStory.js:141]: this is module exports

**Comment by argshook** [comment]
For clarity:

This PR changes the way `story.js` files are doctored during build:

```diff
-export default story({ ... })
+ story({ ... })
```

previously `export default` or `module.exports` object would be wrapped
with `story()` which handles all the storybook registration etc.

no one ever used the exported values of `export default`, what matters
are the side effects of `story()`

Thus, removing `export default` or `module.exports` should be a safe change and more in line with storybook 6 CSF

File: packages/wix-storybook-utils/src/loader/prepareStory/prepareStory.js (hunk 2 above)
[the remark this answers is not in the document]
Comment (argshook) [reply packages/wix-storybook-utils/src/loader/prepareStory/prepareStory.js:141]: oops, i missed the `path.remove()` before, sorry!

**Review by argshook — APPROVED** [review]

**Pull request merged**